HowManyBalloonsAlg.py

```diff
@@ -0,0 +1,15 @@
+"""How Many Balloons Alghorithm"""
+
+import math
+
+class BalloonCount(object):
+    def __init__(self, weight):
+        self.weight = weight
+
+    def how_many_balloons(self):
+        balloons = self.weight * 453.593 / 14
+        return math.ceil(balloons)
+
+weight = BalloonCount(int(input("Enter your weight ")))
```

Comment:
For this is that a specific line, or can you give me an example of how it should look in that code?